src/credentials/CHIPCert.h

```diff
@@ -252,6 +252,15 @@ class ChipDN
      **/
     CHIP_ERROR GetCertChipId(uint64_t & certId) const;
 
+    /**
+     * @brief Retrieve the Fabric ID of a CHIP certificate.
+     *
+     * @param certId  A reference to the certificate Fabric ID value.
+     *
+     * @return Returns a CHIP_ERROR on error, CHIP_NO_ERROR otherwise
```

Comment:
Could you rephrase the comment to focus on non-obvious bits? e.g. any CHIP_ERROR return obviously means 'CHIP_NO_ERROR on success, some error otherwise' and `GetCertFabricId` obviously means 'get the fabric ID from the certitifcate'.

Reply:
Updated.